### ISSUE TITLE
Researsh on delta as a target

### ISSUE THREAD

**Issue by efimovgk** [efimovgk/AutoPrice#8]
It's needed to estimate how new target (price/average price for group) works.
If it doesn't - not need to implement it in further

**Comment by efimovgk** [comment]
Reduced features: ['f01', 'f02', 'f13', 'f27', 'f32', 'f28', 'f19', 'f11', 'f31', 'f30', 'f26', 'f21', 'f23', 'f10', 'f22', 'f16', 'f15', 'f25', 'f24', 'f34', 'f14', 'f17', 'f20', 'f35', 'f50', 'f36', 'f33', 'f57', 'f52', 'f60', 'f40', 'f39', 'f05', 'f38', 'f41', 'f18', 'f71', 'f37', 'f48', 'f44', 'f47', 'f55', 'f49', 'f43', 'f67']

Model params: {'base_score': 0.5,
 'colsample_bylevel': 1,
 'colsample_bytree': 0.8,
 'gamma': 0.0,
 'learning_rate': 0.01,
 'max_delta_step': 0,
 'max_depth': 5,
 'min_child_weight': 3,
 'missing': None,
 'n_estimators': 46200,
 'nthread': -1,
 'objective': 'reg:linear',
 'reg_alpha': 0.1,
 'reg_lambda': 1,
 'scale_pos_weight': 1,
 'seed': 42,
 'silent': True,
 'subsample': 0.9}
